src/createRoutineCreator.js

```diff
@@ -0,0 +1,24 @@
+import toCamelCase from 'camelcase';
+
+import createRoutine from './createRoutine';
+import createActionCreator from './createActionCreator';
+
+export default function createRoutineCreator(stages = [], payloadCreator, metaCreator){
+  const stagesArray = typeof stages === 'string' ? [stages] : stages;
+
+  return function (prefix) {
+    const routine = createRoutine(prefix, payloadCreator, metaCreator);
```

Comment:
[I prefer not to extend routines but build it by using list of stages](https://github.com/afitiskin/redux-saga-routines/issues/56#issuecomment-501991069)